resources/dicts/relationship_events/normal_interactions/trust/decrease.json

```diff
@@ -1,207 +1,237 @@
+			"m_c accuses r_c of not taking care of cats properly.",
+			"m_c doesn't bother to ask r_c for help with a minor injury, worried that {PRONOUN/r_c/subject} might make it worse.",
+			"m_c complains that r_c doesn't seem to understand signs from StarClan very well.",
+			"m_c feels worse after r_c gave {PRONOUN/m_c/object} the wrong treatment for something minor."
+		],
+		"random_status_constraint": [
+			"medicine cat"
+		]
+	},
+	{
+		"id": "trust_for_medcat_de_high2",
+		"intensity": "high",
+		"interactions": [
+			"m_c is dissatisfied that r_c can't seem to get rid of {PRONOUN/m_c/poss} chronic pain.",
+			"m_c is upset that r_c seems to ignore {PRONOUN/r_c/object} more than the other cats.",
+			"m_c thinks that r_c prioritizes other cat's issues instead of m_c's chronic ones."
+		],
+		"main_status_constraint": [
+			"elder"
+		],
+		"random_status_constraint": [
+			"medicine cat"
+		]
+	},
+	{
+		"id": "trust_for_medcat_de_high3",
+		"intensity": "high",
+		"interactions": [
+			"m_c accuses r_c of treating cats incorrectly on purpose!",
+			"m_c saw r_c bringing deathberries into the medicine den.",
+			"r_c was rough with m_c during a recent treatment.",
+			"m_c isn't sure why r_c became a medicine cat at all since {PRONOUN/r_c/subject} {VERB/r_c/seem/seems} to not like helping any cats.",
+			"m_c wouldn't be surprised if r_c faked signs from StarClan..."
+		],
+		"random_status_constraint": [
+			"medicine cat"
+		],
+		"random_trait_constraint": [
+			"cold",
+			"bloodthirsty",
+			"vengeful"
+		]
+	},
+	{
+		"id": "trust_for_medcat_de_high4",
+		"intensity": "high",
+		"interactions": [
+			"m_c is shocked when r_c casually admits forgetting a basic treatment.",
+			"m_c growls frustratingly at r_c when {PRONOUN/r_c/subject} {VERB/r_c/don't/doesn't} pay attention to m_c's health concerns... again.",
+			"m_c looks for r_c for treatment on a minor wound, only to find {PRONOUN/r_c/object} playing with catmint!",
+			"m_c hopes to never have a serious illness with r_c in charge of treatments...",
+			"m_c complains that r_c never takes {PRONOUN/r_c/poss} role of medicine cat seriously."
+		],
+		"random_status_constraint": [
+			"medicine cat"
+		],
+		"random_trait_constraint": [
+			"oblivious",
+			"childish"
+		]
+	},
+	{
+		"id": "trust_for_leader_de_high1",
+		"intensity": "high",
+		"interactions": [
+			"m_c cowers away from r_c after r_c barks an order at {PRONOUN/m_c/object}.",
+			"After hearing r_c's most recent demands, m_c is concerned about the future of the Clan.",
+			"m_c doesn't believe that r_c has the Clan's best interests in mind.",
+			"m_c doesn't understand how StarClan could grant a cat as cold and uncaring as r_c nine lives.",
+			"When it comes down to it, m_c doesn't think {PRONOUN/m_c/subject} could follow r_c's every order."
+		],
+		"random_status_constraint": [
+			"leader"
+		],
+		"random_trait_constraint": [
+			"cold",
+			"bloodthirsty",
+			"vengeful"
+		]
+	}
+]
```

Comment:
throwing this at the bottom; there are a lot of missing lines in here that haven't been rewritten. what is the reasoning for this? /genq

Reply:
Copy-pasting my response from a few posts above!

Some of them I moved from one file to another, some of them I rewrote for clarity or split into different interactions, and some of them I most likely removed on accident while I was going back to clean up lines according to Scribble's original advice. I can cross-reference and make sure that lines I deleted are present in one way or another!!

Sorry about that!!!

Reply:
all good! we weren't sure if it was intentional or not, so figured it was good to check :]